modules/etcd-overview.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="etcd-overview_{context}"]
+= Overview of etcd
+
+etcd is a consistent, distributed key-value store that holds small amounts of data that can fit entirely in memory. Although etcd is a core component of many projects, it is the primary data store for Kubernetes, which is the standard system for container orchestration.
+
+[id="etcd-benefits_{context}"]
+== Benefits of using etcd
+
+By using etcd, you can enjoy several benefits:
+
+* Maintain consistent uptime for your cloud-native applications, and keep them working even if individual servers fail
+* Store and replicate all cluster states for Kubernetes
+* Distribute configuration data to provide redundancy and resiliency for the configuration of nodes
```

Comment:
I _think_ the bullets need periods. I find the ISG confusing when it comes to punctuation in lists.

Reply:
Since all of the bullets are fragments and not complete sentences, I think we can omit the periods. This is the source I checked: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#lists